projects/chaos/src/main.c

```diff
@@ -56,59 +51,76 @@ int main(void) {
     .fault_event = CHAOS_EVENT_CAN_FAULT,
     .tx = { GPIO_PORT_A, 12 },
     .rx = { GPIO_PORT_A, 11 },
-    .loopback = true,
+    .loopback = false,
   };
   can_init(&can_settings, &s_can_storage, s_rx_handlers, SIZEOF_ARRAY(s_rx_handlers));
-  can_register_rx_handler(SYSTEM_CAN_MESSAGE_OVUV_DCDC_AUX, prv_handle_uvov, NULL);
+  can_ack_init(&s_can_ack_requests);
```

Comment:
What's this for? The CAN module has its own ACK request storage.

Reply:
Nvm I got this from a very specialized test.